src/engine/x86-64/X86_64Interpreter.v3

```diff
@@ -2139,18 +2147,29 @@ class X86_64InterpreterGen(ic: X86_64InterpreterCode, w: DataWriter) {
 		decrementVsp();
 		endHandler();
 	}
+	def genV128LoadZero(opcode: Opcode,
+		asm_extract_r_s_i: (X86_64Gpr, X86_64Xmmr, u8) -> X86_64Assembler,
+		asm_insert_s_r_i: (X86_64Xmmr, X86_64Gpr, u8) -> X86_64Assembler) {
+		bindHandler(opcode);
+		load_memarg128(r_xmm0, r_tmp0, r_tmp1); // load memarg to xmm0
```

Comment:
I think this has the same issue as instructions we looked at today: you can't load all 16 bytes because it's possible that that would go off the end of memory and cause a trap.